05.log_and_register_fe_model.py

```
@@ -0,0 +1,185 @@
+# Databricks notebook source
+# MAGIC %pip install /Volumes/main/default/file_exchange/denninger/nyc_taxi-0.0.1-py3-none-any.whl
+
+# COMMAND ----------
+
+dbutils.library.restartPython() 
```

Ruff

Undefined name `dbutils`

(F821)